### ISSUE TITLE
[IDEA] Adding extra data to groups to add more resort POWER 😄 

### ISSUE THREAD

**Issue by bleuge** [unexpectedpanda/retool#239]
Never seen clone lists so accurate :D And other tries was done before

I wonder if additional data could be added to groups (could be scrapped easily from Mobygames, etc). In special I am thinking in game types, years, companys, basic stuff. 

So, with some filtering commands, you could resort your collection, having the great advantages of 1G1R. For example, get NES games with class Shootemup, from Konami, etc ... Or get 1G1R but just racing games from snes and megadrive, etc

I am not any good in python 😄 , but I think the hardwork you already did it, this could be very nice to have as feature.

**Comment by unexpectedpanda** [comment]
I've considered extra filtering, although that comes with the caveat that not all sources agree on what genre a game might be, for example.

So far I've mostly looked at the Launchbox DB. The developer makes a plain old ZIP available for people to use, which might be a bit easier to manage.

**Comment by bleuge** [comment]
Yes there are no "standards" about game genres, styles, etc... I like mobygames way of doing it, but that's just me, also we have screen scraper .fr , etc  
Even the data could be grabbed using skyscraper in retropie (or the gui's scraper), and then just gamelist.xml data linked by title with your DBs. 

Sets should be somewhat statics nowadays?, so this could be a 1 time effort, and if we have the data, filtering and resorting the files could be great!. 

Thanks if you consider it!

**Comment by unexpectedpanda** [comment]
Oh sets are definitely not static, especially not at No-Intro :)

It's very likely I'll have to incorporate some sort of fuzzy matching filter to find titles in a DB to match things up.

Let's call this a long term stretch goal. It's certainly not something I'll implement in the short term, that's dedicated to cleaning up v2 and updating clone lists.

**Comment by bleuge** [comment]
Great!, if you need help with string fuzzy matching, I've tested some time ago, and also the usual pre clean stuff is very important, eliminate articles, convert roman numerals to digits, etc...
